lib/cucumber/http/http_steps.rb

```diff
@@ -8,22 +8,22 @@
 Given /^I send "(.*?)"$/ do |content_type|
   steps %Q{
     Given I set headers:
-      | Content-Type | application/#{content_type} |
+      | Content-Type | #{content_type} |
```

Comment:
This seems like a better approach indeed, since not all media types start with `application/`